devices/adeo.js

```diff
@@ -51,4 +54,25 @@ module.exports = [
         description: 'ENKI Lexman E27 LED white',
         extend: extend.light_onoff_brightness_colortemp({colorTempRange: [153, 370]}),
     },
+    {
+        zigbeeModel: ['LDSENK02F'],
+        model: 'LDSENK02F',
+        description: "10A/16A EU smart plug",
+        vendor: "ADEO",
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report, fz.tuya_switch_power_outage_memory],
+        toZigbee: [tz.on_off, tz.tuya_switch_power_outage_memory],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'haElectricalMeasurement']);
+            endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 100, multiplier: 1});
+            endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {
+            acVoltageMultiplier: 1, acVoltageDivisor: 1, acCurrentMultiplier: 1, acCurrentDivisor: 1000, acPowerMultiplier: 1,
+            acPowerDivisor: 1,
+            });
+        },
+        exposes: [e.switch(), e.power(), e.current(), e.voltage().withAccess(ea.STATE),
+            e.energy(), exposes.enum('power_outage_memory', ea.STATE_SET, ['on', 'off', 'restore'])
```

Comment:
Have you checked that the power outage memory works? (asking because you are using `tuya_switch_power_outage_memory` while this does not look like a TuYa device).

Reply:
I removed that.
I dont use the power information, but I see that the power doesn't work.

![Capture d’écran 2021-08-10 à 09 58 41](https://user-images.githubusercontent.com/86953781/128830496-9f3bbbb6-b11e-4f04-a96d-7c2596cae4b4.png)

Reply:
I made some changes, I have the power now.

![Capture d’écran 2021-08-10 à 10 12 05](https://user-images.githubusercontent.com/86953781/128832232-7965f6d8-f302-4130-92c6-e95eb6d598ff.png)

Reply:
Are you sure the device support voltage and current measurements?

Reply:
Hmmm, there is no information on that.
This device probably don't support this, I removed both of them :)